### DISCUSSION TITLE
Cannot install ddtailer

### DISCUSSION THREAD

**Discussion by srgank67** [anapnoe/stable-diffusion-webui-ux#63] [Q&A] [answered]
Hi! I'm trying to install ddtailer but I have the following error:
![image](https://user-images.githubusercontent.com/106804067/232851543-e25d7d80-1e55-4f3a-b2d4-e35504b4810b.png)
Is there any way to install this extension?

**Comment by anapnoe** [comment, marked as answer]
you need to manually download the extension extract it in the extensions folder 
then you have to edit the file ddtailer.py and replace the lines from 48 to 50
```
run(f'"{python}" -m pip install -U openmim==0.3.7', desc="Installing openmim", errdesc="Couldn't install openmim")
run(f'"{python}" -m mim install mmcv-full==1.7.1', desc=f"Installing mmcv-full", errdesc=f"Couldn't install mmcv-full")
run(f'"{python}" -m mim install mmdet==2.28.2', desc=f"Installing mmdet", errdesc=f"Couldn't install mmdet")
```
restart the ui and let it finish the installation
